plugins/notifications/splunk/README.md

```diff
@@ -0,0 +1 @@
+# cs-splunk-plugin
```

Comment:
Do you want to write documentation for each plugin in crowdsec repo or add doc on crowdsec documentation repo ?

Reply:
I think crowdsec documentation repo would be better.